example/src/App.js

```diff
@@ -74,6 +78,11 @@ export default function App() {
           {text}
         </Latex>
       </div>
+      <div>
+        <h2>Other examples</h2>
```

Comment:
Can you change this this "Examples using JS expressions in LaTeX formula", also display the source code in the example that's used to render this, design is not that important?

Perhaps we can add an example which is using a template string instead of multiple react children.

Reply:
You should have permissions to edit the PR, but I can indeed change the title and add the source later this week!